test/CMakeLists.txt

```
@@ -249,6 +249,24 @@ function(add_perf_test NAME)
 endfunction()
 
 set(IMPLICITGEMM_ARGS ${MIOPEN_TEST_FLOAT_ARG})
+
+set(IMPLICITGEMM_MLIR_COMMON
+    MIOPEN_DEBUG_GCN_ASM_KERNELS=0
+    MIOPEN_DEBUG_CONV_FFT=0
+    MIOPEN_DEBUG_CONV_DIRECT=0
+    MIOPEN_DEBUG_CONV_GEMM=0
+    MIOPEN_DEBUG_CONV_SCGEMM=0
+    MIOPEN_FIND_MODE=1)
```

```suggestion
    MIOPEN_FIND_MODE=normal)
```
When MIOPEN_DEBUG_FIND_ONLY_SOLVER is used, disabling of other solvers or algorithms is redundant.

---

What about `MIOPEN_DEBUG_CONV_GEMM=0`? I think this is still necessary to disable the gemm fallback path.

---

If you do not use Immediate mode, then there is none "Immediate mode fallback path" ;)

---

Hah, thanks for the tips. Will get rid of those redundant environment varaibles.